src/ecrc-api/src/main/resources/application.properties

```diff
@@ -41,7 +42,7 @@ ecrc.get-next-session-id-uri=/rest/VCRC/Source/GetNextSessionId/Services?OrgTick
 ecrc.update-service-financial-txn-uri=/rest/VCRC/Source/UpdateServiceFinancialTxn/Services%s
 ecrc.create-new-crc-service-uri=/rest/VCRC/Source/CreateNewCRCService/Services%s
 ecrc.get-service-fee-amount-uri=/rest/VCRC/Source/GetServiceFeeAmount/Services?OrgTicketNumber=%s&ScheduleTypeCd=%s&ScopeLevelCd=%s
-ecrc.create-applicant-uri=/rest/VCRC/Source/CreateApplicantEx/Services%s
+ecrc.create-applicant-uri=/rest/VCRC/Source/CreateApplicantEx/Services
```

Comment:
missing % after 'ecrc.create-applicant-uri=/rest/VCRC/Source/CreateApplicantEx/Services'  ?

Reply:
This is removed for the path parameter in the web client.